doc/source/_templates/layout.html

```diff
@@ -1,4 +1,8 @@
 {% extends "!layout.html" %}
 
-{% block relbar2 %}{% endblock %}
+{% block extrahead %}
+<link href='http://fonts.googleapis.com/css?family=Open+Sans:400italic,700italic,400,700' rel='stylesheet' type='text/css'>
```

Comment:
yeah it's the only place this google-related line occurs in our code, but this is the template file so eventually it gets automatically put into all of our doc files